dnf-behave-tests/fixtures/specs/dnf-ci-provides-alternatives/bar-1.0.spec

```diff
@@ -0,0 +1,23 @@
+%undefine _debuginfo_subpackages
+
+Name:           bar
+Version:        1.0
+Release:        1
+
+License:        Public Domain
```

Comment:
Is there a reason why bar-1.0.spec URL, License and %install differ from foo.spec? Could you make them the same, so it's clear in how the two spec files differ?

Reply:
Good point, I will unify it.